zimui/src/components/channel/tabs/ShortsTab.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+import { useMainStore } from '@/stores/main'
+import { computed } from 'vue'
+
+import ShortsGridTab from './ShortsGridTab.vue'
+
+
+const main = useMainStore()
+const shortsAvailable = computed(() => main.channel?.userShortUploadsPlaylist)
+</script>
+
+<template>
+  <div v-if="shortsAvailable">
```

Comment:
Do we need this? Short tab is anyway not shown when there is not short, isn't it?